packaging/windows/sharedframework/bundle.wxs

```diff
@@ -42,6 +42,7 @@
                        ProductName="$(var.Crt_ProductName)" 
                        Size="$(var.Crt_Size)" 
                        Version="$(var.Crt_Version)" />
+        <ExitCode Behavior="success" Value="1638" />
```

Comment:
Burn should not try to install higher version at all. Detect Condition ( https://github.com/gkhanna79/core-setup/blob/2f91187544afc8d39a14f1e181fde394accf1bf2/packaging/windows/sharedframework/bundle.wxs#L32) is expected to return success and skip installation of the item ( in this case VC Redist ). Detection Condition need to be looked deeper in this case.